umh-core/pkg/cpuhealth/cgroup_source.go

```
@@ -87,59 +87,94 @@ func (c *cgroupSource) advanceUsageRate(ts time.Time, usage diagnosis.Reading) d
 	if fields[0] == "max" {
 		// Uncapped is a definite no-limit: present, but never a positive capacity.
-		return diagnosis.Known(0.0)
+		return quotaRead{Limit: diagnosis.Known(0.0), Raw: raw}, ReadOK
 	}
 
 	quota, err := strconv.ParseInt(fields[0], 10, 64)
 	if err != nil {
-		return diagnosis.Unknown()
+		return quotaRead{Limit: diagnosis.Unknown(), Raw: raw}, classifyRead(errUnparsableRead)
 	}
 	period, err := strconv.ParseInt(fields[1], 10, 64)
```

golangci-lint (2.13.2)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)